chapters/devsandqueues.txt

```diff
@@ -102,6 +102,17 @@ include::{generated}/api/structs/VkPhysicalDeviceProperties.txt[]
     device.
     See <<sparsememory-physicalprops,Sparse Properties>> for details.
 
+An implementation must: return pname:apiVersion version that is greater or
+equal to 1.0.0, and less than 2.0.0.
```

Comment:
Discussed in the WG today. We think the implementation should be required to return a valid version number, so we'd prefer to say (in the 1.2 spec) that the implementation must: return 1.0.x, 1.1.x, or 1.2.x. If we do that, the note below should also be extended to note that future versions of Vulkan may return higher major or minor release version numbers, and applications should be written to allow for that.

Reply:
Right. Strictly speaking a 1.2 spec should say the device must return exactly 1.2. Though I am not extreme to write it like that. But that also means the device is allowed to be any 1.x version (as long as instance is >1.0). E.g. in 1.1 instance the device can report 1.2. So it feels like 1.1 spec should still say it can be 1.2.